### DISCUSSION TITLE
Supported Superset versions with PostgreSQL

### DISCUSSION THREAD

**Discussion by GOKULAET** [apache/superset#18777] [Q&A / Help]
Hi,
We need to know the compatible versions of superset and PostgreSQL. 

Currently, We run superset of version 1.3.0 and postgresql of version 1.10.
We plan to upgrade postgreSQL to 1.14. Will both be compatible?

**Comment by villebro** [comment]
@GOKULAET I assume you mean versions 10 and 14, not 1.10 and 1.14? Unfortunately I don't believe we directly certify any version of Postgres, but in the docker configs we use Postgres 10. Having said that, since Superset is using very basic Postgres functionality when running the metadata backend, I don't expect to see any problems moving from 10 to 14. So I would consider it safe. If you do try it out, please let us know how it worked out.

**Reply by GOKULAET** [reply]
Okay @villebro
